docs/md/modules-arrays.md

```diff
@@ -80,47 +80,69 @@ To give more information, here are some example inputs to the above endpoints:
     hitting this endpoint without them will return all workloads. However, they cannot be specified
     together.
 
-**GET /api/v1/workload/create**
+**GET /api/v1/workload/{uuid}/workflows**
 
 === "Request"
 
     ```shell
-    curl -X "POST" "http://localhost:8080/api/v1/create" \
-        -H 'Authorization: Bearer '$(gcloud auth print-access-token) \
-        -H 'Accept: application/json' \
-        -H 'Content-Type: application/json' \
-        -d '{
-                "executor": "https://cromwell-gotc-auth.gotc-dev.broadinstitute.org",
-                "output":   "gs://broad-gotc-dev-wfl-ptc-test-outputs/aou-test-output/",
-                "project":  "Example Project",
-                "pipeline": "AllOfUsArrays"
-            }'
+    curl 'http://localhost:8080/api/v1/workload/00000000-0000-0000-0000-000000000000/workflows' \
+        -H 'Authorization: Bearer '$(gcloud auth print-access-token) 
     ```
 
-**GET /api/v1/workload/start**
+**POST /api/v1/workload/create**
```

Comment:
Is this path supposed to be `/api/v1/create` to match the high-level endpoint directory above?  I see this happening similarly for `start`, `stop`, and `append_to_aou`

Reply:
yep. Thanks.